src/CGB_Registers.md

```diff
@@ -90,7 +90,7 @@ be read as "1". Stopping the transfer doesn't set HDMA1-4 to \$FF.
 
 ::: warning WARNING
 
-HBlank DMA should not be started (write to FF55) during a HBlank
+HBlank DMA should not be started (write to FF55) during an HBlank
```

Comment:
It's arguably a weird/confusing/tricky rule, but "a" should be used before a sounded "H", not "an".

Reply:
User voxel#9080 in the gbdev server on Discord raised a question about "an aitch blank" (in the lect of the child actress in the ["A, B, C, D, PCP" anti-drug PSA](https://www.youtube.com/watch?v=Wob_38j5JV8)) vs. "a haitch blank" (in voxel's lect). Surprisingly, ["Haitch or aitch? How a humble letter was held hostage by historical haughtiness" by Kate Burridge and Catherine McBride](https://theconversation.com/haitch-or-aitch-how-a-humble-letter-was-held-hostage-by-historical-haughtiness-97184) argues for a shift toward "haitch" on grounds that children learn letters whose name starts with the sound that it makes more quickly.

Reply:
Because of the pronunciation differences, both "a" and "an" would be correct in this context, of course.

Reply:
I changed it back to "a" to avoid unnecessary commits when both usages are arguably correct. The article was really interesting!

Reply:
Thank you for the feedback ^_^ marked as resolved.